README.md

```diff
@@ -240,6 +240,14 @@ If you are interested in helping make Elyra better, we encourage you to take a l
 [Development Workflow](https://elyra.readthedocs.io/en/latest/developer_guide/development-workflow.html)
 documentation, and invite you to attend our weekly dev community meetings.
 
+### Daily Dev Meetings
+**When**: Every Weekday except Thursdays at [8:30AM Pacific](https://www.thetimezoneconverter.com/?t=8%3A30%20am&tz=San%20Francisco&)
```

Comment:
```suggestion
**When**: Every weekday except Thursdays at [8:30AM Pacific](https://www.thetimezoneconverter.com/?t=8%3A30%20am&tz=San%20Francisco&)
```

Should we also add a little intro statement similar to the community meeting item (for consistency's sake more than anything)?  Perhaps something like:

> Join us for our daily scrum (except Thursdays) to discuss development items you're working on or have questions about.  Everyone is welcome and participation is optional.

Reply:
done!